File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        var temp = 0;
        var ans = 0;
        for (var i = 0; i < nums.Length; ++i) {
            if (nums[i] == 1) {
                temp++;
                ans = Math.Max(ans, temp);
            }
            else
                temp = 0;
        }
        return ans;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.go
func findMaxConsecutiveOnes(nums []int) int {
	maxi := 0
	temp := 0
	for i := 0; i < len(nums); i++ {
		if nums[i] == 1 {
			temp++
			maxi = max(maxi, temp)
		} else {
			temp = 0
		}
	}
	return maxi
}